SConstruct

```diff
@@ -169,7 +169,7 @@ opts.Add(EnumVariable("bits", "Target platform bits", "default", ("default", "32
 opts.Add(EnumVariable("float", "Floating-point precision", "default", ("default", "32", "64")))
 opts.Add(EnumVariable("optimize", "Optimization type", "speed", ("speed", "size", "none")))
 opts.Add(BoolVariable("production", "Set defaults to build Godot for use in production", False))
-opts.Add(BoolVariable("use_lto", "Use link-time optimization", False))
+opts.Add(EnumVariable("lto", "Link-time optimization (for production builds)", "none", ("none", "thin", "full")))
```

Comment:
"for production builds" sounds a bit dogmatic. Perhaps it would be better to write "slow, for production builds" to at least explain why it's only used in these cases? I know that it also makes debugging useless, but I can see LTO being used for development purposes too like benchmarking and whatnot.

Reply:
That's just a hint for when to use it, it doesn't mean you can't use it for other use cases. I think it's fine as is personally.

Reply:
No worries, this stuff's subjective (and also a huge nitpicking). Still, thinking about it the style's consistent with the rest of the messages, so I'll mark this as resolved since it actually makes sense to keep it as is IMO.